Q/QuantRegBuilder/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "QuantRegBuilder"
+version = v"0.1.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/fogarty-ben/QuantReg.jl-Builder.git", "486b3d9ce5f5a0b3c1172c77a47166c57f1cd501")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd QuantReg.jl-Builder/
+mkdir -p ${libdir}
+gfortran -fPIC -shared -std=legacy rqbr.f -o ${libdir}/rqbr.${dlext} 
```

Comment:
Just as a convention, libraries have usually names that start with "lib", so this would be "librqbr", and the same for the next line

Reply:
Got it, changing right now!